init/common.js

```diff
@@ -0,0 +1,175 @@
+const config = require("./config");
+const Strapi = require("@strapi/strapi");
+const fs = require("fs");
+const _ = require("lodash");
+
+const setupEnv = (cb) => {
+  fs.writeFile(".env", convertToEnv(config.env), cb);
+};
+
+const convertToEnv = (object) => {
+  let envFile = "";
+  for (const key of Object.keys(object)) {
+    envFile += `${key}=${object[key]}\n`;
+    if (key === 'NODE_ENV' && process.env.NODE_ENV === 'test') {
```

Comment:
Can you comment why we skip `NODE_ENV` in test?

Reply:
Added